ui/pages/confirmations/components/confirm/info/approve/revoke-static-simulation/revoke-static-simulation.tsx

```diff
@@ -0,0 +1,53 @@
+import { NameType } from '@metamask/name-controller';
+import { TransactionMeta } from '@metamask/transaction-controller';
+import React from 'react';
+import { ConfirmInfoRow } from '../../../../../../../components/app/confirm/info/row';
+import Name from '../../../../../../../components/app/name';
+import { Box } from '../../../../../../../components/component-library';
+import { Display } from '../../../../../../../helpers/constants/design-system';
+import { useI18nContext } from '../../../../../../../hooks/useI18nContext';
+import { useConfirmContext } from '../../../../../context/confirm';
+import StaticSimulation from '../../shared/static-simulation/static-simulation';
+
+export const RevokeStaticSimulation = () => {
+  const t = useI18nContext();
+
+  const { currentConfirmation: transactionMeta } = useConfirmContext() as {
+    currentConfirmation: TransactionMeta;
+  };
+
+  const simulationElements = (
+    <>
+      <ConfirmInfoRow label={t('spendingCap')}>
```

Comment:
Do we not have these two rows defined somewhere already that we can re-use?

Or we could start now and define them centrally somewhere as stand-alone rows for use elsewhere?